cmd/clusterctl/hack/create-local-repository.py

```diff
@@ -167,6 +169,9 @@ def create_local_repositories():
     assert providerList is not None, 'invalid configuration: please define the list of providers to override'
     assert len(providerList)>0, 'invalid configuration: please define at least one provider to override'
 
+    if len(sys.argv) == 1:
+        execCmd(['make', 'kustomize'])
```

Comment:
Why are we now additionally running make kustomize in this case?

As far as I can tell the only time we use it we run it directly before that (l.201-202)

But I'm probably missing something

Reply:
before we were calling make customize several times, one for each provider, now we are calling it once, and only if we are actually going to use it later

Reply:
Should we then drop l.201?

Reply:
@fabriziopandini ^^